src/api/services/timecardService.spec.js

```diff
@@ -30,112 +32,81 @@ const TENANT_ID_PARAM = {
-      undefined
-    );
-  });
+      const response = await getTimecard();
 
-  test('getTimecard throws useful error containing throwing service and function', async () => {
-    api.get.mockImplementation(() => {
-      throw new Error('xyz');
+      expect(api.get).toHaveBeenCalledWith(
+        expect.stringContaining('api/v1/timecard'),
+        undefined
+      );
+      expect(response.data[0].meta).toBeDefined();
+      expect(response.data[0].items).toBeDefined();
+      expect(response.data[0].items.length).toBeGreaterThan(0);
+      expect(response.data).toStrictEqual(timecard);
     });
 
-    try {
-      const response = await getTimecard();
-      expect(response).toBeUndefined();
-    } catch (error) {
-      expect(error).toBeDefined();
-      expect(error.message).toContain('Timecard Service');
-      expect(error.message).toContain('getTimecard');
-      expect(error.message).toContain('xyz');
-    }
-  });
+    it('should use correct endpoint api/v1/timecard', async () => {
+      api.get.mockImplementation(() => Promise.resolve({ data: timecard }));
 
-  test('getTimePeriodTypes uses correct endpoint resources/time-period-type', async () => {
-    api.get.mockImplementation(() =>
-      Promise.resolve({ data: MOCK_TIME_PERIOD_TYPES })
-    );
+      await getTimecard();
 
-    const timePeriodTypes = await getTimePeriodTypes(TENANT_ID_PARAM);
+      expect(api.get).toHaveBeenCalledWith(
+        expect.stringContaining('api/v1/timecard'),
+        undefined
+      );
+    });
 
-    expect(api.get).toHaveBeenCalledWith(
-      expect.stringContaining('resources/time-period-type'),
-      TENANT_ID_PARAM
-    );
-    expect(timePeriodTypes.data.items.length).toBe(7);
-  });
+    it('should throw useful error containing throwing service and function', async () => {
+      api.get.mockImplementation(() => {
+        throw new Error('xyz');
+      });
 
-  test('getTimePeriodTypes throws useful error containing throwing service and function', async () => {
-    api.get.mockImplementation(() => {
-      throw new Error('xyz');
+      try {
+        const response = await getTimecard();
+        expect(response).toBeUndefined();
+      } catch (error) {
+        expect(error).toBeDefined();
+        expect(error.message).toContain('Timecard Service');
+        expect(error.message).toContain('getTimecard');
+        expect(error.message).toContain('xyz');
+      }
     });
-
-    try {
-      const response = await getTimePeriodTypes(TENANT_ID_PARAM);
-      expect(api.get).toHaveBeenCalledWith(
-        expect.stringContaining('resources/time-period-type'),
-        TENANT_ID_PARAM
-      );
-      expect(response).toBeUndefined();
-    } catch (error) {
-      expect(error).toBeDefined();
-      expect(error.message).toContain('Timecard Service');
-      expect(error.message).toContain('getTimePeriodTypes');
-      expect(error.message).toContain('xyz');
-    }
   });
 
-  describe('getTimeEntries', () => {
```

Comment:
Where the `getTimeEntries` tests removed by mistake?

Reply:
They were, re-added them